applications/SwimmingDEMApplication/custom_utilities/L2_error_projection_utility.h

```diff
@@ -119,14 +135,14 @@ double GetL2ScalarProjection(ModelPart& r_model_part)
             for (unsigned int j = 0; j < NumNodes; ++j){
                 error += rGeom[j].FastGetSolutionStepValue(SCALAR_ERROR) * N[j];
             }
-            result += std::pow(error,2) * DetJ[gss] * IntegrationPoints[gss].Weight();
+            result += element_area * std::pow(error,2) * DetJ[gss] * IntegrationPoints[gss].Weight();
```

Comment:
I think this is wrong... You are integrating the error. Why are you multiplying by the 'element area'? The Gauss Points already have that information.

Reply:
Yes, it should be removed.